### ISSUE TITLE
[Issue]: 10.8.0 upgrade breaks; cannot start - permissions and  not start - permissions issue

### ISSUE THREAD

**Issue by cegfault** [jellyfin/jellyfin#7881]
### Please describe your bug

Upgrading to 10.8.0 simply via `apt upgrade` cause a break, where jellyfin could no longer start.  From journalctl (see full message below):
```
jellyfin.service: Changing to the requested working directory failed: Permission denied
jellyfin.service: Failed at step CHDIR spawning /usr/bin/jellyfin: Permission denied
```

Okay, so chdir or chroot jail doesn't have the right permissions. so I found `/var/lib/jellyfin` was owned by `jellyfin:adm`, but files/directories underneath are owned by `jellyfin:jellyfin`, and I see "User" and "Group" in `/lib/systemd/system/jellyfin.service` are both "jellyfin", so let's try that:

```bash
chown jellyfin:jellyfin /var/lib/jellyfin
systemctl start jellyfin
```

Still errors out, but this time it's different (see full message dump below):
```
Unhandled Exception
System.UnauthorizedAccessException: Access to the path '/etc/jellyfin/logging.default.json' is denied.
```

Interesting, so I see `/etc/jellyfin/logging.default.json` is owned by `jellyfin:root`, along with several other files in `/etc/jellyfin`, and the directory is owned by `jellyfin:adm`, so let's try that:

```bash
chown jellyfin:jellyfin /etc/jellyfin /etc/jellyfin/logging.default.json
systemctl start jellyfin
```

and... nope. Now I get:
```bash
 [INF] Application directory: /usr/lib/jellyfin/bin/
 [INF] Applying migration 'CreateNetworkConfiguration'
 [INF] Migration 'CreateNetworkConfiguration' applied successfully
 [FTL] Unhandled Exception
System.UnauthorizedAccessException: Access to the path '/etc/jellyfin/migrations.xml' is denied.
  ---> System.IO.IOException: Permission denied
    --- End of inner exception stack trace ---
```

This process continues for some time. Now I'm getting (full logs below):
```
 [19:58:04] [INF] Jellyfin version: 10.8.0
 [19:58:04] [INF] Environment Variables: ["[JELLYFIN_FFMPEG_OPT, --ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_DATA_DIR, /var/lib/jellyfin]", "[JELLYFIN_LOG_DIR, /var/log/jellyfin]", "[JELLYFIN_WEB_OPT, -
 [19:58:04] [INF] Arguments: ["/usr/lib/jellyfin/bin/jellyfin.dll", "--webdir=/usr/share/jellyfin/web", "--restartpath=/usr/lib/jellyfin/restart.sh", "--ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg", ""]
 [19:58:04] [INF] Operating system: Linux
 [19:58:04] [INF] Architecture: X64
 [19:58:04] [INF] 64-Bit Process: True
 [19:58:04] [INF] User Interactive: True
 [19:58:04] [INF] Processor count: 4
 [19:58:04] [INF] Program data path: /var/lib/jellyfin
 [19:58:04] [INF] Web resources path: /usr/share/jellyfin/web
 [19:58:04] [INF] Application directory: /usr/lib/jellyfin/bin/
 [19:58:04] [INF] Setting cache path: /var/cache/jellyfin
 [19:58:04] [INF] Loading assemblies
 [19:58:04] [ERR] Failed to load assembly /var/lib/jellyfin/plugins/Auto Organize_9.0.0.0/Emby.AutoOrganize.dll. This error occurs when a plugin references an incompatible version of one of the shared libraries. D
 System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
 Method 'ExecuteAsync' in type 'Emby.AutoOrganize.Core.OrganizerScheduledTask' from assembly 'Emby.AutoOrganize, Version=9.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
 Could not load type 'SQLitePCL.pretty.IResultSetValue' from assembly 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
    at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
    at System.Reflection.Assembly.GetTypes()
    at Emby.Server.Implementations.Plugins.PluginManager.LoadAssemblies()+MoveNext()
```

Okay, plugins issue.
```bash
mv /var/lib/jellyfin/plugins /var/lib/jellyfin/plugins.bak
systemctl start jellyfin
```

And now I get:
```
 [20:05:41] [FTL] Error while starting server.
 Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 8: 'attempt to write a readonly database'.
```

Not entirely sure what to do about that one.......

TL;DR there's something funky going on with this upgrade, and it's broken my entire install. Jellyfin won't start.

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Debian 10 (buster)
- Virtualization: LXC
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
Initially I was getting this in systemd:

-- Subject: A start job for unit jellyfin.service has finished successfully
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- A start job for unit jellyfin.service has finished successfully.
-- 
-- The job identifier is 12753.
Jun 11 19:15:11 media systemd[109837]: jellyfin.service: Changing to the requested working directory failed: Permission denied
Jun 11 19:15:11 media systemd[109837]: jellyfin.service: Failed at step CHDIR spawning /usr/bin/jellyfin: Permission denied
```

And then this:
```
Jun 11 19:28:14 media systemd[1]: Started Jellyfin Media Server.
-- Subject: A start job for unit jellyfin.service has finished successfully
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- A start job for unit jellyfin.service has finished successfully.
-- 
-- The job identifier is 13163.
Jun 11 19:28:14 media jellyfin[109942]: Unhandled Exception
Jun 11 19:28:14 media jellyfin[109942]: System.UnauthorizedAccessException: Access to the path '/etc/jellyfin/logging.default.json' is denied.
Jun 11 19:28:14 media jellyfin[109942]:  ---> System.IO.IOException: Permission denied
Jun 11 19:28:14 media jellyfin[109942]:    --- End of inner exception stack trace ---
Jun 11 19:28:14 media jellyfin[109942]:    at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
Jun 11 19:28:14 media jellyfin[109942]:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
Jun 11 19:28:14 media jellyfin[109942]:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
Jun 11 19:28:14 media jellyfin[109942]:    at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
Jun 11 19:28:14 media jellyfin[109942]:    at System.IO.Strategies.FileStreamHelpers.ChooseStrategy(FileStream fileStream, String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
Jun 11 19:28:14 media jellyfin[109942]:    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
Jun 11 19:28:14 media jellyfin[109942]:    at Jellyfin.Server.Program.InitLoggingConfigFile(IApplicationPaths appPaths)
Jun 11 19:28:14 media jellyfin[109942]:    at Jellyfin.Server.Program.StartApp(StartupOptions options)
Jun 11 19:28:14 media jellyfin[109942]:    at Jellyfin.Server.Program.<Main>(String[] args)
Jun 11 19:28:14 media jellyfin[109942]: Unhandled exception. System.UnauthorizedAccessException: Access to the path '/etc/jellyfin/logging.default.json' is denied.
Jun 11 19:28:14 media jellyfin[109942]:  ---> System.IO.IOException: Permission denied
Jun 11 19:28:14 media jellyfin[109942]:    --- End of inner exception stack trace ---
Jun 11 19:28:14 media jellyfin[109942]:    at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
Jun 11 19:28:14 media jellyfin[109942]:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
Jun 11 19:28:14 media jellyfin[109942]:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
Jun 11 19:28:14 media jellyfin[109942]:    at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
Jun 11 19:28:14 media jellyfin[109942]:    at System.IO.Strategies.FileStreamHelpers.ChooseStrategy(FileStream fileStream, String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
Jun 11 19:28:14 media jellyfin[109942]:    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
Jun 11 19:28:14 media jellyfin[109942]:    at Jellyfin.Server.Program.InitLoggingConfigFile(IApplicationPaths appPaths)
Jun 11 19:28:14 media jellyfin[109942]:    at Jellyfin.Server.Program.StartApp(StartupOptions options)
Jun 11 19:28:14 media jellyfin[109942]:    at Jellyfin.Server.Program.<Main>(String[] args)
Jun 11 19:28:14 media systemd[1]: jellyfin.service: Main process exited, code=killed, status=6/ABRT
-- Subject: Unit process exited
```

And finally:
```
-- 
-- The unit jellyfin.service completed and consumed the indicated resources.
Jun 11 19:45:47 media systemd[1]: Started Jellyfin Media Server.
-- Subject: A start job for unit jellyfin.service has finished successfully
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- A start job for unit jellyfin.service has finished successfully.
-- 
-- The job identifier is 14397.
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Jellyfin version: 10.8.0
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Environment Variables: ["[JELLYFIN_CACHE_DIR, /var/cache/jellyfin]", "[JELLYFIN_ADDITIONAL_OPTS, ]", "[JELLYFIN_LOG_DIR, /var/log/jellyfin]", "[JELLYFIN_ARGS, $JELLYFIN_WEB_OPT $JELLYFIN_RESTART_
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Arguments: ["/usr/lib/jellyfin/bin/jellyfin.dll", "--webdir=/usr/share/jellyfin/web", "--restartpath=/usr/lib/jellyfin/restart.sh", "--ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg", ""]
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Operating system: Linux
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Architecture: X64
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] 64-Bit Process: True
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] User Interactive: True
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Processor count: 4
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Program data path: /var/lib/jellyfin
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Web resources path: /usr/share/jellyfin/web
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Application directory: /usr/lib/jellyfin/bin/
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Applying migration 'CreateNetworkConfiguration'
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [INF] Migration 'CreateNetworkConfiguration' applied successfully
Jun 11 19:45:48 media jellyfin[110276]: [19:45:48] [FTL] Unhandled Exception
Jun 11 19:45:48 media jellyfin[110276]: System.UnauthorizedAccessException: Access to the path '/etc/jellyfin/migrations.xml' is denied.
Jun 11 19:45:48 media jellyfin[110276]:  ---> System.IO.IOException: Permission denied
Jun 11 19:45:48 media jellyfin[110276]:    --- End of inner exception stack trace ---
Jun 11 19:45:48 media jellyfin[110276]:    at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
Jun 11 19:45:48 media jellyfin[110276]:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
Jun 11 19:45:48 media jellyfin[110276]:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
Jun 11 19:45:48 media jellyfin[110276]:    at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
Jun 11 19:45:48 media jellyfin[110276]:    at System.IO.Strategies.FileStreamHelpers.ChooseStrategy(FileStream fileStream, String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
Jun 11 19:45:48 media jellyfin[110276]:    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
Jun 11 19:45:48 media jellyfin[110276]:    at Emby.Server.Implementations.Serialization.MyXmlSerializer.SerializeToFile(Object obj, String file)
Jun 11 19:45:48 media jellyfin[110276]:    at Jellyfin.Server.Migrations.MigrationRunner.<>c__DisplayClass3_0.<RunPreStartup>b__1(MigrationOptions options)
Jun 11 19:45:48 media jellyfin[110276]:    at Jellyfin.Server.Migrations.MigrationRunner.PerformMigrations(IMigrationRoutine[] migrations, MigrationOptions migrationOptions, Action`1 saveConfiguration, ILogger logger)
Jun 11 19:45:48 media jellyfin[110276]:    at Jellyfin.Server.Migrations.MigrationRunner.RunPreStartup(ServerApplicationPaths appPaths, ILoggerFactory loggerFactory)
Jun 11 19:45:48 media jellyfin[110276]:    at Jellyfin.Server.Program.StartApp(StartupOptions options)
Jun 11 19:45:48 media jellyfin[110276]:    at Jellyfin.Server.Program.<Main>(String[] args)
```

and finally:
```
-- Subject: Process /usr/bin/jellyfin could not be executed
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- 
-- The job identifier is 16365.
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Jellyfin version: 10.8.0
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Environment Variables: ["[JELLYFIN_FFMPEG_OPT, --ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_DATA_DIR, /var/lib/jellyfin]", "[JELLYFIN_LOG_DIR, /var/log/jellyfin]", "[JELLYFIN_WEB_OPT, -
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Arguments: ["/usr/lib/jellyfin/bin/jellyfin.dll", "--webdir=/usr/share/jellyfin/web", "--restartpath=/usr/lib/jellyfin/restart.sh", "--ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg", ""]
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Operating system: Linux
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Architecture: X64
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] 64-Bit Process: True
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] User Interactive: True
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Processor count: 4
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Program data path: /var/lib/jellyfin
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Web resources path: /usr/share/jellyfin/web
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Application directory: /usr/lib/jellyfin/bin/
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Setting cache path: /var/cache/jellyfin
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [INF] Loading assemblies
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [ERR] Failed to load assembly /var/lib/jellyfin/plugins/Auto Organize_9.0.0.0/Emby.AutoOrganize.dll. This error occurs when a plugin references an incompatible version of one of the shared libraries. D
Jun 11 19:58:04 media jellyfin[110727]: System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Jun 11 19:58:04 media jellyfin[110727]: Method 'ExecuteAsync' in type 'Emby.AutoOrganize.Core.OrganizerScheduledTask' from assembly 'Emby.AutoOrganize, Version=9.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
Jun 11 19:58:04 media jellyfin[110727]: Could not load type 'SQLitePCL.pretty.IResultSetValue' from assembly 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
Jun 11 19:58:04 media jellyfin[110727]:    at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
Jun 11 19:58:04 media jellyfin[110727]:    at System.Reflection.Assembly.GetTypes()
Jun 11 19:58:04 media jellyfin[110727]:    at Emby.Server.Implementations.Plugins.PluginManager.LoadAssemblies()+MoveNext()
Jun 11 19:58:04 media jellyfin[110727]: System.TypeLoadException: Method 'ExecuteAsync' in type 'Emby.AutoOrganize.Core.OrganizerScheduledTask' from assembly 'Emby.AutoOrganize, Version=9.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an im
Jun 11 19:58:04 media jellyfin[110727]: System.TypeLoadException: Could not load type 'SQLitePCL.pretty.IResultSetValue' from assembly 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
Jun 11 19:58:04 media jellyfin[110727]: [19:58:04] [FTL] Error while starting server.
Jun 11 19:58:04 media jellyfin[110727]: System.UnauthorizedAccessException: Access to the path '/var/lib/jellyfin/plugins/Auto Organize_9.0.0.0/meta.json' is denied.
Jun 11 19:58:04 media jellyfin[110727]:  ---> System.IO.IOException: Permission denied
Jun 11 19:58:04 media jellyfin[110727]:    --- End of inner exception stack trace ---
Jun 11 19:58:04 media jellyfin[110727]:    at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
Jun 11 19:58:04 media jellyfin[110727]:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
```
```


### FFmpeg logs

_No response_

### Please attach any browser or client logs here

_No response_

### Please attach any screenshots here

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by miceZipper** [comment]
+1

**Comment by microtechno9000** [comment]
Also have this issue, upgraded via apt yesterday and now Jellyfin is unresponsive with permission errors as above
Ubuntu 20.04.4 LTS

**Comment by brimur** [comment]
Just did a clean install of Jellyfin and I am getting this error too

`Fetched 137 MB in 45s (3,043 kB/s)
Selecting previously unselected package jellyfin-server.
(Reading database ... 155422 files and directories currently installed.)
Preparing to unpack .../0-jellyfin-server_10.8.0-1_amd64.deb ...
Unpacking jellyfin-server (10.8.0-1) ...
Selecting previously unselected package jellyfin-web.
Preparing to unpack .../1-jellyfin-web_10.8.0-1_all.deb ...
Unpacking jellyfin-web (10.8.0-1) ...
Selecting previously unselected package libllvm11:amd64.
Preparing to unpack .../2-libllvm11_1%3a11.0.0-2~ubuntu20.04.1_amd64.deb ...
Unpacking libllvm11:amd64 (1:11.0.0-2~ubuntu20.04.1) ...
Selecting previously unselected package libxcb-randr0:amd64.
Preparing to unpack .../3-libxcb-randr0_1.14-2_amd64.deb ...
Unpacking libxcb-randr0:amd64 (1.14-2) ...
Selecting previously unselected package jellyfin-ffmpeg5.
Preparing to unpack .../4-jellyfin-ffmpeg5_5.0.1-6-focal_amd64.deb ...
Unpacking jellyfin-ffmpeg5 (5.0.1-6-focal) ...
Selecting previously unselected package jellyfin.
Preparing to unpack .../5-jellyfin_10.8.0-1_all.deb ...
Unpacking jellyfin (10.8.0-1) ...
Setting up jellyfin-web (10.8.0-1) ...
Setting up jellyfin-server (10.8.0-1) ...
Setting up libllvm11:amd64 (1:11.0.0-2~ubuntu20.04.1) ...
Setting up libxcb-randr0:amd64 (1.14-2) ...
Setting up jellyfin-ffmpeg5 (5.0.1-6-focal) ...
Setting up jellyfin (10.8.0-1) ...
Processing triggers for systemd (245.4-4ubuntu3.17) ...
Processing triggers for libc-bin (2.31-0ubuntu9.9) ...
BigPlex ~: service jellyfin status
● jellyfin.service - Jellyfin Media Server
     Loaded: loaded (/lib/systemd/system/jellyfin.service; enabled; vendor preset: enabled)
    Drop-In: /etc/systemd/system/jellyfin.service.d
             └─jellyfin.service.conf
     Active: failed (Result: exit-code) since Wed 2022-06-15 11:59:12 UTC; 6min ago
    Process: 3002 ExecStart=/usr/bin/jellyfin ${JELLYFIN_WEB_OPT} ${JELLYFIN_RESTART_OPT} ${JELLYFIN_FFMPEG_OPT} ${JELLYFIN_SERVICE_OPT} ${JELLYFIN_NOWEBAPP_OPT} ${JELLYFIN_ADDITIONAL_OPTS} (code=exited, status=200/CHDIR)
   Main PID: 3002 (code=exited, status=200/CHDIR)

Jun 15 11:59:12 BigPlex systemd[3002]: jellyfin.service: Failed at step CHDIR spawning /usr/bin/jellyfin: Permission denied
Jun 15 11:59:12 BigPlex systemd[1]: jellyfin.service: Main process exited, code=exited, status=200/CHDIR
Jun 15 11:59:12 BigPlex systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Jun 15 11:59:12 BigPlex systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 6.
Jun 15 11:59:12 BigPlex systemd[1]: Stopped Jellyfin Media Server.
Jun 15 11:59:12 BigPlex systemd[1]: jellyfin.service: Start request repeated too quickly.
Jun 15 11:59:12 BigPlex systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Jun 15 11:59:12 BigPlex systemd[1]: Failed to start Jellyfin Media Server.
BigPlex ~: chmod 777 -R /usr/bin/jellyfin
BigPlex ~: service jellyfin start
BigPlex ~: service jellyfin status
● jellyfin.service - Jellyfin Media Server
     Loaded: loaded (/lib/systemd/system/jellyfin.service; enabled; vendor preset: enabled)
    Drop-In: /etc/systemd/system/jellyfin.service.d
             └─jellyfin.service.conf
     Active: failed (Result: exit-code) since Wed 2022-06-15 12:09:20 UTC; 1s ago
    Process: 3254 ExecStart=/usr/bin/jellyfin ${JELLYFIN_WEB_OPT} ${JELLYFIN_RESTART_OPT} ${JELLYFIN_FFMPEG_OPT} ${JELLYFIN_SERVICE_OPT} ${JELLYFIN_NOWEBAPP_OPT} ${JELLYFIN_ADDITIONAL_OPTS} (code=exited, status=200/CHDIR)
   Main PID: 3254 (code=exited, status=200/CHDIR)

Jun 15 12:09:20 BigPlex systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 5.
Jun 15 12:09:20 BigPlex systemd[1]: Stopped Jellyfin Media Server.
Jun 15 12:09:20 BigPlex systemd[1]: jellyfin.service: Start request repeated too quickly.
Jun 15 12:09:20 BigPlex systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Jun 15 12:09:20 BigPlex systemd[1]: Failed to start Jellyfin Media Server.
`

**Comment by crobibero** [comment]
Likely related to https://github.com/jellyfin/jellyfin/issues/7503

Should be fixed by https://github.com/jellyfin/jellyfin/pull/7887

**Comment by microtechno9000** [comment]
Tried the solution listed in #7503, although that hasn't resolved the issue with 10.8.0

`sudo systemctl status jellyfin
● jellyfin.service - Jellyfin Media Server
     Loaded: loaded (/lib/systemd/system/jellyfin.service; enabled; vendor preset: enabled)
    Drop-In: /etc/systemd/system/jellyfin.service.d
             └─jellyfin.service.conf
     Active: failed (Result: exit-code) since Tue 2022-06-21 13:58:07 ACST; 11min ago
    Process: 216190 ExecStart=/usr/bin/jellyfin ${JELLYFIN_WEB_OPT} ${JELLYFIN_RESTART_OPT} ${JELLYFIN_FFMPEG_OPT} >
   Main PID: 216190 (code=exited, status=200/CHDIR)

Jun 21 13:58:07 maxwell.wilson systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 5.
Jun 21 13:58:07 maxwell.wilson systemd[1]: Stopped Jellyfin Media Server.
Jun 21 13:58:07 maxwell.wilson systemd[1]: jellyfin.service: Start request repeated too quickly.
Jun 21 13:58:07 maxwell.wilson systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Jun 21 13:58:07 maxwell.wilson systemd[1]: Failed to start Jellyfin Media Server.
`

`cat /var/log/syslog | grep jellyfin
Jun 21 13:43:54 maxwell systemd[213023]: jellyfin.service: Changing to the requested working directory failed: Permission denied
Jun 21 13:43:54 maxwell systemd[213023]: jellyfin.service: Failed at step CHDIR spawning /usr/bin/jellyfin: Permission denied
Jun 21 13:43:54 maxwell systemd[1]: jellyfin.service: Main process exited, code=exited, status=200/CHDIR
Jun 21 13:43:54 maxwell systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Jun 21 13:43:54 maxwell systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 1.
Jun 21 13:43:54 maxwell systemd[213036]: jellyfin.service: Changing to the requested working directory failed: Permission denied
Jun 21 13:43:54 maxwell systemd[213036]: jellyfin.service: Failed at step CHDIR spawning /usr/bin/jellyfin: Permission denied
`

**Comment by microtechno9000** [comment]
@cegfault have you updated to 10.8.1?
I just did on Ubuntu 20.04 and it has not resolved the permissions issue. Keen to see if it has for anyone else.

**Comment by rsvcoder** [comment]
I had similar issue; but it was not exactly the same.
jellyfin.service: Changing to the requested working directory failed: No such file or directory

It was above.
SO I followed these steps to resolve mine:

created these folders manually:
/var/lib/jellyfin
/var/log/jellyfin
/var/cache/jellyfin
/var/lib/jellyfin/data
/var/lib/jellyfin/metadata

Then following commands:
sudo apt purge jellyfin-web
sudo apt update
sudo apt install jellyfin-web
sudo systemctl restart jellyfin.service

Give permissions:
chown jellyfin /var/lib/jellyfin
chown jellyfin /var/log/jellyfin
chown jellyfin /var/cache/jellyfin
chown jellyfin /var/lib/jellyfin/data
chown jellyfin /var/lib/jellyfin/metadata

Mine started working with this.

**Comment by koningcool** [comment]
Same for me on latest version, lets hope it gets fixed soon!

**Comment by AshleighTheCutie** [comment]
I'm having this issue too, but it randomly occurred last night with nothing being changed. It first appeared as a database access error, and then spiraled worse from there.

**Comment by synkoptik** [comment]
> I had similar issue; but it was not exactly the same. jellyfin.service: Changing to the requested working directory failed: No such file or directory
> 
> It was above. SO I followed these steps to resolve mine:
> 
> created these folders manually: /var/lib/jellyfine /var/log/jellyfin /var/cache/jellyfine /var/lib/jellyfin/data /var/lib/jellyfin/metadata
> 
> Then following commands: sudo apt purge jellyfin-web sudo apt update sudo apt install jellyfin-web sudo systemctl restart jellyfin.service
> 
> Give permissions: chown jellyfin /var/lib/jellyfin chown jellyfin /var/log/jellyfin chown jellyfin /var/cache/jellyfin chown jellyfin /var/lib/jellyfin/data chown jellyfin /var/lib/jellyfin/metadata
> 
> Mine started working with this.

This worked for me.. just fix the typo on "jellyfine" to jellyfin
